README-sycl.md

```diff
@@ -227,16 +226,15 @@ Similarly, user targeting Nvidia GPUs should expect at least one SYCL-CUDA devic
 source /opt/intel/oneapi/setvars.sh
 
 # Build LLAMA with MKL BLAS acceleration for intel GPU
-mkdir -p build && cd build
 
 # Option 1: Use FP32 (recommended for better performance in most cases)
-cmake .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx
+cmake -B build -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx
 
 # Option 2: Use FP16
-cmake .. -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
+cmake -B build -DLLAMA_SYCL=ON -DCMAKE_C_COMPILER=icx -DCMAKE_CXX_COMPILER=icpx -DLLAMA_SYCL_F16=ON
 
-#build all binary
-cmake --build . --config Release -j -v
+# build all binary
+cmake --build build -j -v
```

Comment:
```suggestion
cmake --build . --config Release -j -v
```
Thank you to enhance the build. 
I test it to work well for SYCL backend on Intel GPU.
I suggest keeping the old format, which is explicit to show how to build Release. And user know how to build with Debug mode easily.
This is guide. We should show easy and more info to user as possible.

Reply:
Thanks for testing this out (and for catching the other typo!!)

The old format is actually misleading as ‘—config Debug’ would also build in Release mode (only builds Debug for multi config generators such as Visual Studio, XCode or Ninja Multi-Config, not for Makefile or normal Ninja modes). To build in debug *in general*, you need to set ‘-DCMAKE_BUILD_TYPE=Debug’ on the first “config” cmake command. I’ve hinted about this in the main readme but maybe I should expand a bit / add this to each of the readmes?

Reply:
Reverted the `--config Release` flag (as it's not the default for MSVC / Xcode etc) and documented how to do Debug builds on the main Readme.